Replace start using a service screenshots (#4170)
alphagov/govuk-design-system: src/patterns/start-using-a-service/index.md

![A screenshot showing an example of a start point within a multipart guide. The page is shown as part of a set of pages and its body text explains what the user will need to use the service, followed by a start button labelled 'Apply now'.](multipart-guide-start-point.png)

> You can read more about the benefits of providing a start page within a multipart guide in [this blog post by DWP's digital team talking about how they improved the Carer's Allowance guide](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).

Reviewer: ```suggestion
[Read a blog post about how DWP's digital team found that providing a start page within a multipart guide for Carer's Allowance improved the journey for users](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
```
